src/Lachain.Core/ValidatorStatus/ValidatorStatusManager.cs

```diff
@@ -384,8 +385,10 @@ private void SubmitAttendanceDetection()
 
         private void AddTxToPool(Transaction tx)
         {
-            var receipt = _transactionSigner.Sign(tx, _privateWallet.EcdsaKeyPair);
-            _sendingTxHash = tx.FullHash(receipt.Signature);
+            var useNewChainId =
+                HardforkHeights.IsHardfork_6Active(_stateManager.LastApprovedSnapshot.Blocks.GetTotalBlockHeight());
```

Comment:
Should it be GetTotalBlockHeight() + 1 ? because in pool it is +1 : https://github.com/LATOKEN/lachain/blob/chain_id_change/src/Lachain.Core/Blockchain/Pool/TransactionPool.cs#L178